.vortex/installer/src/Downloader/RepositoryDownloader.php

```
@@ -412,4 +391,24 @@ protected function validateLocalRefExists(string $repo, string $ref): void {
+   * Build the headers sent with every request to the repository host.
+   *
+   * @param array<string, string> $headers
+   *   Additional headers to include.
+   *
+   * @return array<string, string>
+   *   The headers, authorised when a token is available.
+   */
+  protected static function requestHeaders(array $headers = []): array {
+    $headers['User-Agent'] = 'Vortex-Installer';
+
+    $github_token = Env::get('GITHUB_TOKEN');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\DrevOps\VortexInstaller\Utils\Env' in method 'requestHeaders'. (undefined)

(StaticAccess)